apps/react/src/shared/components/MyNavbar/MyNavbar.tsx

```diff
@@ -0,0 +1,89 @@
+import { logoutUser } from '@js-camp/react/store/user/dispatchers';
+import { useAppDispatch, useAppSelector } from '@js-camp/react/store/store';
+import { selectUser } from '@js-camp/react/store/user/selector';
+import {
+  Box,
+  AppBar,
+  Toolbar,
+  IconButton,
+  Typography,
+  Menu,
+  MenuItem,
+  Avatar,
+  Tooltip,
+} from '@mui/material';
+import { FC, memo, useState, MouseEvent } from 'react';
+import { Link } from 'react-router-dom';
+
+import styles from './MyNavbar.module.css';
+
+const MyNavbarComponent: FC = () => {
+  const dispatch = useAppDispatch();
+  const user = useAppSelector(selectUser);
+  const [anchorElementUser, setAnchorElementUser] =
+    useState<null | HTMLElement>(null);
```

Comment:
Something weird with formatting.
```suggestion
  const [anchorElementUser, setAnchorElementUser] = useState<null | HTMLElement>(null);
```

Reply:
It's prettier formatting. I just auto format and it turn out like this.